.github/workflows/ci.yaml

```diff
@@ -156,7 +159,9 @@ jobs:
             echo "mark_expression=not unstable" >> "$GITHUB_OUTPUT"
           fi
       - name: Run integration tests
-        run: tox run -e integration -- "${{ matrix.groups.path_to_test_file }}" --group="${{ matrix.groups.group_number }}" -m '${{ steps.select-test-stability.outputs.mark_expression }}' --mysql-router-charm-series=${{ matrix.ubuntu-versions.series }} --mysql-router-charm-bases-index=${{ matrix.ubuntu-versions.bases-index }}
+        if: 
```

Comment:
TIL: What exactly empty `if:` means?

Reply:
removed in ee4b09862143abb00c36232590ac72f383d91e5b. it was a leftover of a previous attempt at something